pkg/sql/logictest/testdata/logic_test/fk

```diff
@@ -3671,3 +3671,52 @@ ALTER TABLE child_59582 ADD FOREIGN KEY (i) REFERENCES child_59582(rowid)
 
 statement ok
 DROP TABLE parent_59582, child_59582
+
+subtest reference_constraint_different_order
+
+# https://github.com/cockroachdb/cockroach/issues/63324
+# Referencing an existing constraint by fields in a different order.
+# Should still find existing constraint.
+
+statement ok
+CREATE TABLE reference_constraint_different_order_parent (
+	a
+		INT8 NOT NULL,
+	b
+		INT8 NOT NULL,
+	c
+		INT8 NOT NULL,
+	d
+		VARCHAR(10)
+);
+CREATE TABLE reference_constraint_different_order_child (
+	id INT8, a INT8, b INT8, c INT8, e VARCHAR(10)
+);
+ALTER TABLE reference_constraint_different_order_parent
+	ADD CONSTRAINT p_pk PRIMARY KEY (a, b, c);
+
+statement ok
+ALTER TABLE reference_constraint_different_order_child
+	ADD CONSTRAINT p_fk FOREIGN KEY (a, b, c) REFERENCES reference_constraint_different_order_parent (a, c, b);
+
+statement ok
```

Comment:
can you insert something that is not in the same order and check that it is errors? look up `statement error` as per above.

Reply:
The INSERT shouldn't error. It's the foreign key field order that is the issue

Reply:
yes, i mean add an entry that would violate the foreign key and prove that it errors with `statement error`

Reply:
Ahhh sorry I misunderstood, will add